PIC_undo_batch.py

```diff
@@ -7,15 +7,24 @@
 from specify_db import SpecifyDb
 import picdb_config
 import traceback
-
-
 class PicturaeUndoBatch(Importer):
     def __init__(self, MD5):
         super().__init__(picturae_config, "Botany")
         self.purge_code = MD5
         self.batch_db_connection = SpecifyDb(db_config_class=picdb_config)
         self.run_all(MD5=self.purge_code)
 
+
+
+    def get_attachment_location(self, timestamp1, timestamp2):
```

Comment:
function added to get  list of attachment locations on server from attachments table between two timestamps, used for removing attachments from server